### DISCUSSION TITLE
TTS Model as a service?

### DISCUSSION THREAD

**Discussion by MrEdwards007** [coqui-ai/TTS#2231] [General Q&A]
An interesting idea that I was searching for but did not find.
Does a service exist where you hand-off your voice data to someone and they hand you back a model that you can use?
What I am referring to is you possess your own model to use in an offline, portable, platform agnostic way.  

I can take my model to Window, Linux, Mac for generating life-like speech. 
This is different than services where you supply your voice and you utilize TTS in an interactive way.

**Comment by erogol** [comment]
Thanks for sharing. We plan to explore this use case in the fullness of time.
